code/_onclick/hud/screen_objects.dm

```diff
@@ -60,6 +60,42 @@
 	var/hand_index
 
 /obj/abstract/screen/holomap
+	icon = 'icons/480x480.dmi'
+	icon_state = "blank"
+
+/obj/abstract/screen/holomap/Click(location,control,params)
+	var/obj/structure/deathsquad_gravpult/G = locate() in get_turf(usr)
+	var/list/params_list = params2list(params)
+	if (params_list.len)
+		G.aim = Clamp(text2num(params_list["icon-y"]), 0, 480)
+		G.update_aim()
+
+
+
+///////////////////////////////////////
+/obj/abstract/screen/interface
+	var/mob/user = null
+	var/obj/machine = null
+	var/action = ""
+
+/obj/abstract/screen/interface/New(turf/loc,u,m,a,i,i_s="",l="CENTER,CENTER",px=0,py=0)
```

Comment:
Okay, do a doc-comment above this proc to say what each of these variables mean, as single-letter variables in a function's gonna be a bitch to maintain by anyone else.

Reply:
Done